src/core/lib/iomgr/tcp_posix.cc

```diff
@@ -1809,7 +1809,8 @@ grpc_endpoint* grpc_tcp_create(grpc_fd* em_fd,
   if (setsockopt(tcp->fd, SOL_TCP, TCP_INQ, &one, sizeof(one)) == 0) {
     tcp->inq_capable = true;
   } else {
-    gpr_log(GPR_DEBUG, "cannot set inq fd=%d errno=%d", tcp->fd, errno);
+    const char* msg = "dbg-info, non-fatal: cannot set inq fd=%d errno=%d";
```

Comment:
I would inline this, instead of defining a local variable.

GPR_DEBUG already implies that this is a debug-log why do we need this?

Reply:
I requested @arjunroy to do this PR since this log message ends up littering the test logs and we've been misinterpreting it as a serious issue

Reply:
Oh I see. Thank you for the context. I'm ok with removing this LOG line all together :-)  It's not critical and we can get the data with ftrace.